.travis.yml

```diff
@@ -1,49 +1,76 @@
-sudo: required
-
-language: generic
+sudo: 'required'
 
 services:
-  - docker
+  - 'docker'
+
+language:
+  - 'bash'
+
+env:
+  global:
+    - NODE_RED_VERSION=x.y.z
+    - TARGET=raymondmm/node-red
+    - QEMU_VERSION=v3.0.0
+  matrix:
+    - DOCKER_FILE=Dockerfile.alpine-tmpl BUILD_FROM=amd64/node NODE_VERSION=10 OS=alpine QEMU_ARCH=x86_64 ARCH=amd64
+    - DOCKER_FILE=Dockerfile.alpine-tmpl BUILD_FROM=arm32v6/node NODE_VERSION=10 OS=alpine QEMU_ARCH=arm ARCH=arm32v6
+    - DOCKER_FILE=Dockerfile.slim-tmpl BUILD_FROM=arm32v7/node NODE_VERSION=10 OS=slim QEMU_ARCH=arm ARCH=arm32v7
```

Comment:
Should we have a full fat version with all the dev/build tools in ? python node-gyp build-essential etc

Reply:
If you can list all tools which might come in handy for a full fat version, i can adapt these in:
- 0.20.7-alpine-amd64-devtools
- 0.20.7-alpine-arm32v6-devtools
- 0.20.7-buster-slim-arm32v7-devtools 
- 0.20.7-alpine-arm64v8-devtools 

And manifest list them in:
- 0.20.7-devtools
- latest-devtools